master.js

```diff
@@ -1007,15 +962,74 @@ async function getPlugins(){
 		let plugin = plugins[i];
 		if(plugin.main.onLoadFinish && typeof plugin.main.onLoadFinish == "function") await plugin.main.onLoadFinish({plugins});
 	}
-	console.log("All plugins loaded in "+(Date.now() - startPluginLoad)+"ms");
 	return plugins;
 }
 
+async function startServer() {
+	// Set the process title, shows up as the title of the CMD window on windows
+	// and as the process name in ps/top on linux.
+	process.title = "clusterioMaster";
+
+	// add better stack traces on promise rejection
+	process.on('unhandledRejection', r => console.log(r));
 
-masterPlugins = await getPlugins();
+	/** Sync */
+	function randomStringAsBase64Url(size) {
+	  return base64url(crypto.randomBytes(size));
+	}
+
+	if (!fs.existsSync("secret-api-token.txt")) {
+		config.masterAuthSecret = randomStringAsBase64Url(256);
+		fs.writeFileSync("config.json",JSON.stringify(config, null, 4));
+		fs.writeFileSync("secret-api-token.txt", jwt.sign({ id: "api" }, config.masterAuthSecret, {
```

Comment:
Like this example - this file name is used in multiple places in the code and cannot be changed unless you edit the code.

Reply:
Another option would be to allow you to generate a token on the web interface.  Or maybe a whole slave config file.  But again.  Out of scope for this PR.  Open an issue about it.